play/tests/unittests/PhaseResults.test.tsx

```diff
@@ -0,0 +1,165 @@
+/** @jest-environment jsdom */
+import React from 'react';
+import { render, waitFor, act } from '@testing-library/react';
+import { ThemeProvider } from '@mui/material/styles';
+import { I18nextProvider } from 'react-i18next';
+import { MemoryRouter } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import ReactModal from 'react-modal';
+import { 
+  ApiClient,
+  Environment,
+  GameSessionParser,
+  IAWSGameSession,
+  IGameSession,
+  IChoice,
+  IQuestion
+} from '@righton/networking';
+import Theme from '../../src/lib/Theme';
+import i18n from './mock/translations/mockTranslations';
+import PhaseResults from '../../src/pages/PhaseResults';
+import mockPhaseOneZeroPointsGameSession from './mock/gamesessions/scoretests/phaseOneZeroPointsGameSession.json';
+import mockPhaseOneTenPointsGameSession from './mock/gamesessions/scoretests/phaseOneTenPointsGameSession.json';
+import mockPhaseTwoUnpopularAnswerGamesession from './mock/gamesessions/scoretests/phaseTwoUnpopularAnswerGameSession.json';
+import mockPhaseTwoPopularAnswerGamesession from './mock/gamesessions/scoretests/phaseTwoPopularAnswerGameSession.json';
+
+ReactModal.setAppElement('body');
+
+// Mock the ApiClient.updateTeam method
+jest.mock('@righton/networking', () => ({
+  ...jest.requireActual('@righton/networking'),
+  ApiClient: jest.fn().mockImplementation(() => ({
+    updateTeam: jest.fn(),
+  })),
+}));
+const apiClient = new ApiClient(Environment.Staging);
```

Comment:
Can this be done in a separate file and then referenced in test files, we should only have one reference to `ApiClient(Environment.Staging)`. If we decide to switch environment or change the `ApiClient` to receive extra parameters, that would break all unit tests and requires going to all files to update them. We can probably use one of the global methods mentioned [here](https://jestjs.io/docs/api)

Reply:
This has been done via our mock of ApiClient.ts